mu-plugins/blocks/language-suggest/language-suggest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Block Name: Language Suggest
+ * Description: A block for use across the whole wp.org network.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\MU_Plugins\wporg;
```

Comment:
This was maybe a mistake from the generator script, but this should probably be `Language_Suggest`

```suggestion
namespace WordPressdotorg\MU_Plugins\Language_Suggest;
```

Reply:
Yeah, I think it is. I got passed me in another review as well :)